### PR TITLE
[google-map-react]: add `libraries` and `region` parameters to `BootstrapURLKeys`

### PR DESCRIPTION
**Pull request by NicoleEtLui**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<https://github.com/google-map-react/google-map-react/blob/b3c27dfb0e323fb2d554112158cc7bbf2fcae347/src/loaders/google_map_loader.js#L68-L82>>
- [issue related](https://github.com/google-map-react/google-map-react/issues/713)

### CODE REVIEW

**Comment by conradob** [comment]
Hi @honzabrecka, can you review this ASAP please?

**Review by androsj — APPROVED** [review]

**Comment by androsj** [comment]
Please also include the `region` property. I think that would complete the options allowed by the load script.

File: types/google-map-react/google-map-react-tests.tsx [hunk 1]
```diff
@@ -3,8 +3,8 @@ import * as React from 'react';
 
 const center = { lat: 0, lng: 0 };
 
-const key: BootstrapURLKeys = { key: 'my-google-maps-key' };
-const client: BootstrapURLKeys = { client: 'my-client-identifier', v: '3.28' , language: 'en' };
+const key: BootstrapURLKeys = { key: 'my-google-maps-key', libraries: "places" };
```
Comment (androsj) [inline types/google-map-react/google-map-react-tests.tsx]:
```suggestion
const key: BootstrapURLKeys = { key: 'my-google-maps-key', libraries: "geometry", region: "GB" };
```

File: types/google-map-react/google-map-react-tests.tsx [hunk 2]
```diff
@@ -3,8 +3,8 @@ import * as React from 'react';
...
+const client: BootstrapURLKeys = { client: 'my-client-identifier', v: '3.28' , language: 'en', libraries: "places" };
```
Comment (androsj) [inline types/google-map-react/google-map-react-tests.tsx]:
```suggestion
const client: BootstrapURLKeys = { client: 'my-client-identifier', v: '3.28' , language: 'en', libraries: "places", region: "PR" };
```

File: types/google-map-react/index.d.ts [hunk 3]
```diff
@@ -6,7 +6,7 @@
 
 import * as React from 'react';
 
-export type BootstrapURLKeys = ({ key: string; } | { client: string; v: string; }) & { language?: string };
+export type BootstrapURLKeys = ({ key: string; } | { client: string; v: string; }) & { language?: string; libraries?: string[] | string; };
```
Comment (androsj) [inline types/google-map-react/index.d.ts]:
```suggestion
export type BootstrapURLKeys = ({ key: string; } | { client: string; v: string; }) & { language?: string; region?: string; libraries?: string[] | string; };
```

**Comment by androsj** [comment]
@NicoleEtLui  Sweet! Thanks for also adding the region prop. Now I guess the pull request needs a new title.

**Review by androsj — APPROVED** [review]

**Comment by NicoleEtLui** [comment]
Hello @androsj, are we waiting for something to merge this PR ? :)

**Comment by androsj** [comment]
@NicoleEtLui I'm not a member of DefinitelyTyped. Someone else must merge it.

**Comment by jessetrinity** [comment]
The reason these are not getting merged is because the definition owner is not approving them. Generally new authors should add themselves (if they want to help maintain the definitions), then they can approve PRs and get them in the Express Merge pile.

@honzabrecka you are on couple of these. Any interest?

**Pull request merged**